Loop/View Models/SimpleBolusViewModel.swift

```diff
@@ -99,6 +104,22 @@ class SimpleBolusViewModel: ObservableObject {
             _manualGlucoseString = newValue
         }
     }
+    
+    private func updateNotice() {
+        let minRecommendationGlucose = displayMealEntry ? LoopConstants.simpleBolusCalculatorMinGlucoseMealBolusRecommendation : LoopConstants.simpleBolusCalculatorMinGlucoseBolusRecommendation
```

Comment:
[nit] are there unit tests that cover this new code?  If so, then 👍

Reply:
There are more unit tests coming in the next PR (which also has more alarms).